rocm-cmake/.SRCINFO

```diff
@@ -1,13 +1,12 @@
 pkgbase = rocm-cmake
 	pkgdesc = CMake modules for common build tasks needed for the ROCm software stack
-	pkgver = 3.10.0
+	pkgver = 4.0.0
 	pkgrel = 1
 	url = https://github.com/RadeonOpenCompute/rocm-cmake
 	arch = x86_64
 	license = MIT
 	makedepends = cmake
-	source = rocm-cmake-3.10.0.tar.gz::https://github.com/RadeonOpenCompute/rocm-cmake/archive/rocm-3.10.0.tar.gz
+	source = rocm-cmake-4.0.0.tar.gz::https://github.com/RadeonOpenCompute/rocm-cmake/archive/rocm-4.0.0.tar.gz
 	sha256sums = 751be4484efdcf0d5fa675480db6e2cddab897de4708c7c7b9fa7adb430b52d7
```

Comment:
Checksum not updated

Reply:
@Erihel This PR has been merged already please open another issue for this.